### DISCUSSION TITLE
小图封面模糊

### DISCUSSION THREAD

**Discussion by lyppiao** [ywmoyue/biliuwp-lite#553] [General]
不知道为什么，小图封面越来越模糊。
包括但不仅限于。稍后再看、历史纪录、播放详情页封面等，越小越糊。
![屏幕截图 2024-03-27 225238](https://github.com/ywmoyue/biliuwp-lite/assets/143381863/99e52c66-45cc-4166-a7e2-8bd849f05f64)

**Comment by GD-Slime** [comment]
好像是和直播间里面原来舰长图标模糊一样的原因。
后来我换另一种图片缩放模式后就较为正常了。
希望能帮到忙 @ywmoyue

**Reply by ywmoyue** [reply]
好的，非常感谢

**Comment by lyppiao** [comment]
好像连用户头像也是糊的
